### ISSUE TITLE
modify_air_speed only affects the vertical speed of creative flight

### ISSUE THREAD

**Issue by Micalobia** [apace100/origins-fabric#705]
What is says on the title, not much more to explain.

**Comment by eggohito** [comment]
Which version of Origins are you using?

**Comment by Micalobia** [comment]
V1.2.1, which is the latest for 1.20.1, not sure if there's a more recent one.

**Comment by Micalobia** [comment]
The way I wrote the power looks like this, if that's relevant at all
I've got the value set to 0 for testing, and you can't go up or down but can still move sideways perfectly.
```json
{
    "type": "origins:modify_air_speed",
    "modifier": {
        "operation": "set_total",
        "value": 0
    },
    "condition": {
        "type": "origins:and",
        "conditions": [
            {
                "type": "origins:gamemode",
                "gamemode": "survival"
            },
            {
                "type": "origins:creative_flying"
            }
        ]
    }
}
```

**Comment by eggohito** [comment]
This should be fixed in [Origins 1.11.x](https://github.com/apace100/origins-fabric/releases/tag/v1.11.0)
